SECURITY.md

```diff
@@ -0,0 +1,19 @@
+# Security Policy
+
+We aim to keep Timescale safe for everyone. 
+Publicly disclosing bugs in a public forum puts everyone in the Timescale community at risk.
```

Comment:
"bugs" can be other things than vulnerabilities. 
Maybe we keep it simple and remove this entirely. We already have the section on how to report a vulnerability.

Reply:
fixed.